e2etests/tests/evpn_l2.go

```diff
@@ -90,108 +91,145 @@ var _ = Describe("Routes between bgp and the fabric", Ordered, func() {
 		secondPod *corev1.Pod
 		nad       nad.NetworkAttachmentDefinition
 	)
-
-	BeforeEach(func() {
+	type testCase struct {
+		firstPodIPs, secondPodIPs, hostARedIPs, hostBRedIPs, l2GatewayIPs []string
+	}
+	DescribeTable("should create two pods connected to the l2 overlay", func(tc testCase) {
```

Comment:
not sure I like having a table-ish test (the list of tests below) inside a table test

Reply:
fair enough, don't think disassembling is worth the effort